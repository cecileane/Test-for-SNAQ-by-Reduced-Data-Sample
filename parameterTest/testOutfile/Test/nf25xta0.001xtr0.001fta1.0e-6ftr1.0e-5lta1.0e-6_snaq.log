optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=1.0e-6,
                       xtolAbs=0.001, xtolRel=0.001.
 max number of failed proposals = 25, liktolAbs = 1.0e-6.
rootname for files: nf25xta0.001xtr0.001fta1.0e-6ftr1.0e-5lta1.0e-6_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 30082
seed: 30082 for run 1
Sat Oct  1 17:30:50 2016

 BEGIN SNaQ for run 1, seed 30082 and hmax 1
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 42 (failures: 25) with -loglik=204.56962 and ht_min=[0.10163,0.40832,0.43013,0.39929,10.0]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 2		 1	 |	 NA 		 NA 		 NA 
mvorigin	 11		 11		 0	 |	 1		 1		 --
mvtarget	 9		 9		 0	 |	 0		 0		 --
chdir		 5		 5		 1	 |	 1		 1		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 15		 10		 4	 |	 2		 2		 --
Total		 42		 37		 6	 |	 4		 4		 3
Proportion	 -- 		 1.0		 0.2	 |	 -- 		 1.0		 0.8

 FINISHED SNaQ for run 1, -loglik of best 204.56962150295834
((((6,(5,#H7:::0.10163114484475448):10.0):0.399286807787548,3):0.4301283450658818,(4)#H7:::0.8983688551552456):0.40831594025644535,2,1);
---------------------
seed: 71534 for run 2
Sat Oct  1 17:30:52 2016

 BEGIN SNaQ for run 2, seed 71534 and hmax 1
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 55 (failures: 25) with -loglik=215.7344 and ht_min=[0.42251,0.11872,0.0,1.00295,2.21139]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 19		 19		 5	 |	 0		 0		 --
mvtarget	 10		 10		 0	 |	 0		 0		 --
chdir		 9		 9		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 16		 9		 1	 |	 1		 1		 --
Total		 55		 48		 7	 |	 1		 1		 1
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 2, -loglik of best 215.734400799311
((1,(6,(5,#H7:::0.422507586972559):2.211393602218833):1.0029487225181302):0.11871973597765732,(4,2):0.0,(3)#H7:::0.577492413027441);
---------------------

Sat Oct  1 17:30:56 2016
MaxNet is ((((6,(5,#H7:::0.10163114484475448):10.0):0.399286807787548,3):0.4301283450658818,(4)#H7:::0.8983688551552456):0.40831594025644535,2,1); 
with -loglik 204.56962150295834
