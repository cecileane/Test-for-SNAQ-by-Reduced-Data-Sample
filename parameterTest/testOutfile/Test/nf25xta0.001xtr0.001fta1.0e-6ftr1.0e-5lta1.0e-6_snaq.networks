((((6,(5,#H7:::0.10163114484475448):10.0):0.399286807787548,3):0.4301283450658818,(4)#H7:::0.8983688551552456):0.40831594025644535,2,1);, with -loglik 204.56962150295834 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
((((6,(5)#H7:::0.9563716137508145):0.39228572265195244,3):0.33119337054499653,(4,#H7:::0.043628386249185525):10.0):0.4848094389804214,2,1);, with -loglik 209.9980130498165
((((2,1):0.42917020632824654)#H7:0.6535091966216422::0.6281264419575833,3):0.16437413632173078,6,(5,(4,#H7:5.703082294260139::0.37187355804241673):2.8569112827444365):0.0);, with -loglik 236.69326006024667
((((6)#H7:::0.5591063759147772,3):0.0,(4,(5,#H7:::0.44089362408522276):1.9019489386843478):0.0):0.626592397750085,2,1);, with -loglik 243.80972436007536
(((3)#H7:::1.0,(4,(5,(#H7:::0.0,6):9.54390789957021):0.46869009348977747):0.0):0.6298930544098448,2,1);, with -loglik 243.81159489911707
