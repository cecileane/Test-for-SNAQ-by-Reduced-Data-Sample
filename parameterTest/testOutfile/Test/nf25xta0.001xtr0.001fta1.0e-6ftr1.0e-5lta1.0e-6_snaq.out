((((6,(5,#H7:::0.10163114484475448):10.0):0.399286807787548,3):0.4301283450658818,(4)#H7:::0.8983688551552456):0.40831594025644535,2,1); -Ploglik = 204.56962150295834
 Dendroscope: ((((6,(5,#H7):10.0):0.399286807787548,3):0.4301283450658818,(4)#H7):0.40831594025644535,2,1);
 Elapsed time: 6.007703139 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 ((((6,(5,#H7:::0.10163114484475448):10.0):0.399286807787548,3):0.4301283450658818,(4)#H7:::0.8983688551552456):0.40831594025644535,2,1);, with -loglik 204.56962150295834
 ((1,(6,(5,#H7:::0.422507586972559):2.211393602218833):1.0029487225181302):0.11871973597765732,(4,2):0.0,(3)#H7:::0.577492413027441);, with -loglik 215.734400799311
-------
